libraries/adaptive-expressions/src/expressionProperties/enumExpression.ts

```diff
@@ -20,13 +20,11 @@ export class EnumExpression<T> extends ExpressionProperty<T> {
     }
 
     public setValue(value: T | string | Expression): void {
-        this.value = undefined;
-        this.expression = undefined;
+        super.setValue(undefined);
```

Comment:
Does this inherit documentation from ExpressionProperty<T>?

Reply:
added doc.